src/main/rom.h

```diff
@@ -124,6 +124,7 @@ typedef struct
    unsigned char savetype;
    unsigned char players; /* Local players 0-4, 2/3/4 way Netplay indicated by 5/6/7. */
    unsigned char rumble; /* 0 - No, 1 - Yes boolean for rumble support. */
+   unsigned char alternate_vi_timing;
```

Comment:
It does fix the game I've tested it. I'd rather someone with push access to the repo added a comment rather than me submitting a pull request just for that since this is already merged